### PR TITLE
New about page narrative fix #15

### LINKED ISSUE #15: "about us" page with icons on the world map
There is an idea to put everyone participating in the project on the map to show how international our team is. It can basically look like the draft 
ppl
For now only icons (no names or any other data, but this is rather discussable). We will be adding everyone who filled the team list on GoogleDoc, however if anyone else wants to join - please write your country and city in the comments below.

### PR DESCRIPTION
**Pull request by MiklerGM**
image

### CODE REVIEW

File: src/pages/About.js [hunk 1]
```diff
@@ -41,17 +41,49 @@ const text = ['We are a community of enthusiasts behind Chron.',
...
 
+const timeline = [{
+  date: '7 March 2017',
+  text: 'Chronist team launched first demo'
+}, {
+  date: '11 November 2017',
+  text: 'First narrative made by Chronist'
+}, {
+  date: '12 January 2018',
+  text: 'Launch of the first slack and the ChronoScio project announced by Ollie Bye'
```
Comment (ataalik) [inline src/pages/About.js]: I would change this a bit to not include slack. Maybe something like 

> Launch of first demo by ChronoScio announced by Ollie Bye.

File: src/pages/About.js [hunk 2]
```diff
@@ -41,17 +41,49 @@ const text = ['We are a community of enthusiasts behind Chron.',
...
+}, {
+  date: '6 July 2018',
+  text: 'ChronoScio team started to work on website'
```
Comment (ataalik) [inline src/pages/About.js]: Well the work for the first demo I launched was made in the start of 2018 and after that we also worked with dwaxe on another iteration with django that we scrapped as the project got bigger. This date, I think, corresponds to the repo we started with Amaury and Liam present for the first time. So, if it doesn't seem too pedantic, we could change this to reflect that I guess.

Comment (MiklerGM) [reply src/pages/About.js]: We could add more steps or change this date.

First commit to [interactivemap](https://github.com/chronoscio/interactivemap/commits/master) repository was on `21 Oct 2017`
For [interactive-map-backend](https://github.com/chronoscio/interactive-map-backend/commits/master) it was on `19 Jan 2018`

Comment (MiklerGM) [reply src/pages/About.js]: Just write the dates and the header for them, I'll add them, no worries

Comment (ataalik) [reply src/pages/About.js]: No need to bloat the narrative with all the small dates I guess. I would just change this line as: 
> With the addition of new members, ChronoScio team started to work on the second iteration of their website.

**Comment by MiklerGM** [comment]
image

**Pull request merged**